src/geometry/circle-circle-intersection.md

```diff
@@ -32,3 +32,5 @@ The only degenerate case we need to consider separately is when the centers of t
 ## Practice Problems
 
 [RadarFinder](https://community.topcoder.com/stat?c=problem_statement&pm=7766)
+[Runaway to a shadow - Codeforces](http://codeforces.com/problemset/problem/681/E)
```

Comment:
Please add newlines between problems, so that they appear one per line, not all in one line. You can use http://e-maxx-eng.appspot.com/test.php to check how the resulting page looks.